Xwt.XamMac/Xwt.Mac/CalendarBackend.cs

```diff
@@ -0,0 +1,137 @@
+﻿//
+// CalendarBackend.cs
+//
+// Author:
+//       Ezequiel Taranto <ezequiel89@gmail.com>
+//		 Claudio Rodrigo Pereyra Diaz <claudiorodrigo@pereyeradiaz.com.ar>
+//
+// Copyright (c) 2015 Hamekoz
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+using System;
+using Xwt.Backends;
+
+#if MONOMAC
+using nint = System.Int32;
+using nfloat = System.Single;
+using MonoMac.Foundation;
+using MonoMac.AppKit;
+
+
+#else
+using Foundation;
+using AppKit;
+#endif
+
+namespace Xwt.Mac
+{
+	public class CalendarBackend: ViewBackend<NSDatePicker,ICalendarEventSink>, ICalendarBackend
+	{
+		public CalendarBackend ()
+		{
+		}
+
+		public override void Initialize ()
+		{
+			base.Initialize ();
+			ViewObject = new MacCalendar ();
+		}
+
+		public override void EnableEvent (object eventId)
+		{
+			base.EnableEvent (eventId);
+			if (eventId is CalendarEvent)
+				Widget.Activated += HandleValueChanged;
+		}
+
+		public override void DisableEvent (object eventId)
+		{
+			base.DisableEvent (eventId);
+			if (eventId is CalendarEvent)
+				Widget.Activated -= HandleValueChanged;
+		}
+
+		void HandleActivated (object sender, EventArgs e)
+		{
+			ApplicationContext.InvokeUserCode (((ICalendarEventSink)EventSink).OnValueChanged);
+		}
+
+		public DateTime Date {
+			get {
+				return (DateTime)Widget.DateValue;
+			}
+			set {
+				var currentDate = (NSDate)value;
```

Comment:
It looks like it is not compatible with Xamarin.Mac (throws System.ArgumentException: DateTimeKind.Unspecified cannot be safely converted). My DatePicker timezone fix doesn't work either, so there is some more work to be done.